setup.cfg

```diff
@@ -24,6 +24,7 @@ install_requires =
     asdf-astropy>=0.3
     asdf>=2.14
     ndcube>=2.0
+    jsonschema<4.18
```

Comment:
Oh no, do we really need this in specutils too? Or is putting at in tox.ini enough?